### PR TITLE
fix images

### PR DESCRIPTION
**Pull request by Mathieudewet**

### CODE REVIEW

**Comment by Mathieudewet** [comment]
Related to https://github.com/api-platform/docs/pull/549

**Comment by Mathieudewet** [comment]
Tested locally with :

```
#!/bin/sh
set -ex
cd "$(dirname "$0")/../"

# Remove previous documentation

rm -fr src/pages/docs
rm -fr src/pages/images

# Checkout documentation

git clone -b fix/images-docs --single-branch https://github.com/Mathieudewet/docs.git src/pages/docs

cp -R src/pages/docs/images public/
cp -R src/pages/docs/images src/pages/

# Generate "nav.yml"

bin/check-outline
bin/generate-nav
```

**Comment by Mathieudewet** [comment]
Travis fail because of https://github.com/api-platform/docs/pull/549 not merged.

**Comment by Mathieudewet** [comment]
Requires rights to make a call to git remote show git@github.com:api-platform/docs.git. in order to figure out which branch is the default one.

File: bin/retrieve-documentation [hunk 1]
```diff
@@ -5,10 +5,14 @@ cd "$(dirname "$0")/../"
 # Remove previous documentation
 
 rm -fr src/pages/docs
+rm -fr src/pages/images
 
 # Checkout documentation
 
-git clone https://github.com/api-platform/docs.git src/pages/docs
+git clone -b `git remote show git@github.com:api-platform/docs.git | grep -e"Branche HEAD \| HEAD branch"| cut -d ":" -f 2` --single-branch https://github.com/api-platform/docs.git src/pages/docs
```
Comment (teohhanhui) [inline bin/retrieve-documentation]: Hmm... French? :confused: So the output is dependent on the user's system language?

Comment (dunglas) [reply bin/retrieve-documentation]: Add ˋLANG=C` before the command to force the lang to be default one.

**Comment by Mathieudewet** [comment]
I'll update it in another PR, let's merge it like that as we'll need to retrieve the default branch in https://github.com/api-platform/website/issues/57 so we'll do that change at the same time.

**Comment by dunglas** [comment]
Closed by #78.

**Pull request closed without merging**